README.md

```diff
@@ -15,9 +15,24 @@ pip install -r requirements.txt
 ```
 pip install -e .
 ```
-3. Generate your api key at https://dataland.com/api-key and put it into [the config file](dataland_analytics/config.py) as `PERSONAL_USER_API_KEY`
 
-4. Execute the analytics scripts found in [this folder](dataland_analytics/analysis_scripts/). Some of them might require additional setup for succesful execution which is described in the script itself
+3. Create and install Python Clients of the Dataland APIs
+   * Install [openapi-generator-cli](https://github.com/OpenAPITools/openapi-generator) on your machine
+   * Navigate to the [datasets folder](api_clients/datasets_api/) and execute 
+
+      ```
+	  openapi-generator-cli generate -g python -i ./datasets_open_api.yaml --additional-properties=packageName=dataland_datasets
+	  ```
+
+   * After generation, change the package `NAME` in the [setup.py](api_clients/datasets_api/setup.py) file from `openapi-client` to `dataland_datasets_client`
```

Comment:
Das mit der Umbenennung verstehe ich nicht ganz. Warum gibt man nicht gleich den richtigen Namen an in der Zeile vorher? Scheinbar wird "_" automatisch zu "-" aber das könnte man ja im Namen ändern das es ohne Umbenennung hier funktioniert.

Reply:
Ah da habe ich das Readme nicht nochmal upgedated. Durch die `--additional-properties` wird dieser Schritt komplett unnötig. Wird entsprechend gelöscht.